feat(PasswordInput): add component (#1745)
gravity-ui/uikit: src/components/controls/PasswordInput/README.md

> ## Password Input

Reviewer: Prepare the file for landing, use `LANDING_BLOCK` `GITHUB_BLOCK` containers. See other readmes as a reference

```tsx
import {PasswordInput} from '@gravity-ui/uikit';
```

> Password Input display component

Reviewer: ```suggestion
`TextInput` for typing passwords and other sensitive information. It can be rendered with copy and reveal buttons for more convinient usage.
```

> 

Reviewer: Let's also make separate sections `Reveal Button` and `Copy Button` describing behavior and props to enable it

> ### PropTypes

Reviewer: ```suggestion
### Properties
```

> Same as [TextInput component](https://github.com/gravity-ui/uikit/blob/main/src/components/controls/TextInput/README.md), with some exceptions:

Reviewer: ```suggestion
`TextInput` [properties](https://github.com/gravity-ui/uikit/blob/main/src/components/controls/TextInput/README.md#properties), with some exceptions and additions:
```

- `value` is required property;
- `onUpdate` is required property;
- `type` is omitted;

> | Property         | Type      | Required | Default | Description                                                                  |

Reviewer: Let's use default header style for such tables:
```
| Name         | Description           |      Type       |     Default     |
| :----------- | :-------------------- | :-------------: | :-------------: |
```
